OpenOversight/app/email.py

```diff
@@ -21,7 +21,7 @@ def send_email(to, subject, template, **kwargs):
     msg.body = render_template(template + ".txt", **kwargs)
     msg.html = render_template(template + ".html", **kwargs)
     # Only send email if we're in prod or staging, otherwise log it so devs can see it
-    if app.env in ("staging", "production"):
+    if app.debug in ("staging", "production"):
```

Comment:
Hi @sea-kelp, I have changed `app.env` to `app.debug`. Is this fine?

Reply:
We actually want to use `app.config["ENVIRONMENT"]` to check the value of the new variable you've added!

Reply:
Ok, makes sense